spec/site_prism/waiter_spec.rb

```diff
@@ -23,5 +23,29 @@
           .with_message(/#{timeout}/)
       end
     end
+
+    context 'with a custom sleep_duration' do
+      let(:timeout) { 0.1 }
+      let(:sleep_duration_long) { 0.5 }
+      let(:sleep_duration_short) { 0.01 }
+
+      # rubocop:disable RSpec/MultipleExpectations
+      # rubocop:disable Style/Semicolon
+      it 'when setting sleep_duration > timeout, error raise and yield execute 2 times' do
+        count = 0
+        expect { described_class.wait_until_true(timeout, sleep_duration_long) { count += 1; false } }
```

Comment:
```
swallow_timeout do
  described_class.wait_until_true(timeout, sleep_duration_long) do
    count += 1
    false
  end
end
```

Reply:
cf: https://github.com/site-prism/site_prism/blob/main/spec/spec_helper.rb#L59-L69

Reply:
Thanks! I'll fix this.

Reply:
I changed code.  https://github.com/site-prism/site_prism/pull/105/commits/e2c4728f4feb468edec730e0acb8835139be9b86